### ISSUE TITLE
Ctrl + T

### ISSUE THREAD

**Issue by KaKi87** [keeweb/keeweb#848]
Hey,

When Keeweb tab is focused, I can't use the Ctrl + T shortcut, which is used to open a new blank tab.

Could you unlock that, or maybe unlock all shortcuts, please ?

Thanks !

**Comment by antelle** [comment]
Hi,
What did you expect from this shortcut? It's used for auto-type, which is supported only in desktop apps.

**Comment by KaKi87** [comment]
Ctrl + T is for creating a new tab in my browser.

**Comment by antelle** [comment]
You mean, it doesn’t open a mew tab if keeweb is focused? Wow that’s weird, browser normally don’t allow to override it, which browser/os is it?

**Comment by KaKi87** [comment]
Vivaldi Browser 1.13.1008.40 x64, Windows 10 16299.192

**Comment by antelle** [comment]
Ok, I’ll look onto it. Probably you should also report it to them, browser must not allow to override this shortcut.

**Comment by KaKi87** [comment]
Okay, thanks

**Comment by dploeger** [comment]
I can also reproduce that on Vivaldi, but only on Vivaldi. Never heard about that before. 😁

**Comment by dploeger** [comment]
Okay, this is a known issue with vivaldi. It always first triggers the web site shortcuts and only, if that isn't handled, triggers the browser shortcut. See here: https://forum.vivaldi.net/topic/23950/keyboard-shortcuts

Please watch the issue there. 

I don't think, this is Keeweb-related. May we close this issue then, @KaKi87?

**Comment by antelle** [comment]
Let's move it, it's better to keep it open to help others find it, since it still exists.

**Comment by KaKi87** [comment]
Well, I gave up...

I actually have a bigger problem (which I gave up on too) : I can't use one of the best feature of Vivaldi, i.e. the panels, because 1. KeeWeb won't allow multiple tabs, and 2. KeeWeb won't save without a pop-up.

So I bypassed that limit by creating a ["shortcut web page"](https://kaki87.net/keeweb/) on which I come back to which I return after each time I use keeweb and I try to not forgot going on a real tab when I need to change data.

This solution works, but it's very annoying.
